service/model/project.go

```diff
@@ -21,30 +21,54 @@ type Project struct {
 	Name                      string
 	SlackChannelID            string
 	ReleaseNotificationConfig ReleaseNotificationConfig
-	GithubRepositoryURL       url.URL
+	GithubRepositoryURL       url.URL // TODO remove this field
 	GithubRepo                *GithubRepo
```

Comment:
I want to keep `GithubRepo` struct in the service model, instead of having these fields  directly in project struct.

```
type Project struct {
...
   URL       url.URL
   OwnerSlug string
   RepoSlug  string
}
```

Why?

- I can easily check if GithubRepo is set `p.GithubRepo != nil`
- I have method for setting repo for project `func (p *Project) SetGithubRepo(repo *GithubRepo) {`
- and mainly github client can return github repo as one struct (instead of returning 3 params):
```
func (c *Client) ReadRepository(ctx context.Context, tkn string, rawRepoURL string) (svcmodel.GithubRepo, error) {
```

Reply:
Yep, sounds fair.